actions/changelog-generate/action.yaml

```diff
@@ -47,7 +47,7 @@ runs:
     - name: "Set tags"
       id: setTags
```

Comment:
why are some of the variables (also action inputs) in camel case 👀 I would prefer following a consistent style

Reply:
you are right. the vars have been updated..